lib/modules/manager/npm/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ParseLockFileResult {
+  detectedIndent: string;
+  lockFileParsed: any;
```

Comment:
Can you please create mor concrete types instead of any or use `unknown` if we don't use that at all.

Reply:
OK, I did try to strict-type the lockfile format because using `unknown` was even messier IMO. Let me know what you think